docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -0,0 +1,449 @@
+    Channel 4: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 5: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 6: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 7: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+
+    Sensor 'gnss':
+    name: uavcan_gnss
+
+    Sensor 'mag':
+    name: uavcan_mag
+
+    Online nodes (Node ID, Health, Mode):
+        5 OK         OPERAT    
+        127 OK         OPERAT    
+
+    uavcan: cycle time: 49750 events, 2398095us elapsed, 48.20us avg, min 13us max 152183us 730.617us rms
+    uavcan: cycle interval: 49750 events, 2701.02us avg, min 26us max 152824us 1225.585us rms
+
+Test Throttle
+##############
+
+Older PX4 Firmware (v1.13.2 and earlier)
+-----------------------------------------
+It is also possible to test spinning the module in this console using the motor_test command. Typing “motor_test -h” in the console will provide help on how to use the command. 
+As a basic example, to spin the motor with an ESC index of 1 at 20% of its maximum, you could use the command “motor_test test -m 1 -p 20”. The image below shows some 
+examples of using the motor_test command.
+
+.. figure:: ../_static/tutorial_images/dronecan_px4_tutorial/mavlink_console_motor_test.png
+    :align: center
+    :width: 60%
+    :alt: Motor Test in MAVLink Console
+
+    Motor Test in MAVLink Console
+
+Newer PX4 Firmware (v1.13.3 and later)
+---------------------------------------
+Integration with the module can be tested using the actuator_test command. Typing "actuator_test" in the console will provide help on how to use the command.
+
+For example, to command motor 1 to spin at 5% for 5 seconds, you can send the command "actuator_test set -m 1 -v 0.05 -t 5". The code block below shows the output
+of the help for the actuator_test command, as well as this example command::
+
+    nsh> actuator_test 
+    Usage: actuator_test <command> [arguments...]
+    Commands:
+
+    set           Set an actuator to a specific output value
+
+    The actuator can be specified by motor, servo or function directly:
+        [-m <val>]  Motor to test (1...8)
+        [-s <val>]  Servo to test (1...8)
+        [-f <val>]  Specify function directly
+        -v <val>    value (-1...1)
+        [-t <val>]  Timeout in seconds (run interactive if not set)
+                    default: 0
+
+    iterate-motors Iterate all motors starting and stopping one after the other
+
+    iterate-servos Iterate all servos deflecting one after the other
+    
+    nsh> actuator_test set -m 1 -v 0.05 -t 5
+
+Virtual Joysticks
+******************
+For a test that is closer to actual flight, the `Virtual Joysticks <https://docs.qgroundcontrol.com/master/en/SettingsView/VirtualJoystick.html>`_ in PX4 can be used to control connected 
+modules while the flight controller is connected to a computer. Enable the virtual joysticks as detailed in the PX4 documentation, and ensure that all Vertiq modules are powered 

```

Comment:
link to how to enable them? or just spell it out here maybe?

Reply:
The virtual joysticks page I already linked you tells you how to enable them as the first thing on there. But I'll put another even more specific link here as well